data/carousel.toml

```diff
@@ -0,0 +1,72 @@
+[[items]]
+title = "Spotlight PA Featured Impact"
+tagline = "Housing"
+text = "A Spotlight PA investigation into Pennsylvania’s pandemic mortgage relief program found that long delays and poor communication were leaving thousands in limbo, with their homes and utilities on the line. The story prompted the state to fire the third-party contractor that was running the program."
+short = "The state fired a third-party contractor after Spotlight PA revealed serious issues with Pennsylvania’s pandemic mortgage relief program."
+link = "https://www.spotlightpa.org/series/the-waiting-game/"
```

Comment:
Make links relative so local dev/a deploy preview doesn't accidentally send you to the real site and then you wonder why it isn't refreshing.